lib/easypost/webhook.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
+# Each Webhook contains the url which EasyPost will notify whenever an object in our system updates. Several types of objects are processed
```

Comment:
Need a period in the end.

Reply:
Nice find. To not dismiss the approvals I'll move forward with this but fix it in a future PR.